### ISSUE TITLE
[Bug]: Missing apcaccess tool in docker

### ISSUE THREAD

**Issue by misaka-eth** [netdata/netdata#12346]
### Bug description

APC UPS monitoring not working in docker container because off missing `apcupsd` package

### Expected behavior

Monitoring shows data after configuration by guide: https://learn.netdata.cloud/docs/agent/collectors/charts.d.plugin/apcupsd

### Steps to reproduce

1. Install netdata in docker
2. Configure APC UPS Monitoring: https://learn.netdata.cloud/docs/agent/collectors/charts.d.plugin/apcupsd
...

### Installation method

docker

### System info

```shell
bash-5.1# uname -a; grep -HvE "^#|URL" /etc/*release
Linux hostname 5.15.10-arch1-1 #1 SMP PREEMPT Fri, 17 Dec 2021 11:17:37 +0000 x86_64 Linux
/etc/alpine-release:3.15.0
/etc/os-release:NAME="Alpine Linux"
/etc/os-release:ID=alpine
/etc/os-release:VERSION_ID=3.15.0
/etc/os-release:PRETTY_NAME="Alpine Linux v3.15"
```

### Netdata build info

```shell
Version: netdata v1.33.1-99-nightly
Configure options:  '--prefix=/usr' '--sysconfdir=/etc' '--localstatedir=/var' '--libexecdir=/usr/libexec' '--libdir=/usr/lib' '--with-zlib' '--with-math' '--with-user=netdata' '--without-bundled-protobuf' '--with-bundled-libJudy' 'CFLAGS=' 'LDFLAGS='
Install type: oci
    Binary architecture: x86_64
Features:
    dbengine:                   YES
    Native HTTPS:               YES
    Netdata Cloud:              YES
    ACLK Next Generation:       YES
    ACLK-NG New Cloud Protocol: YES
    ACLK Legacy:                NO
    TLS Host Verification:      YES
    Machine Learning:           YES
    Stream Compression:         YES
Libraries:
    protobuf:                YES (system)
    jemalloc:                NO
    JSON-C:                  YES
    libcap:                  NO
    libcrypto:               YES
    libm:                    YES
    tcalloc:                 NO
    zlib:                    YES
Plugins:
    apps:                    YES
    cgroup Network Tracking: YES
    CUPS:                    NO
    EBPF:                    NO
    IPMI:                    YES
    NFACCT:                  NO
    perf:                    YES
    slabinfo:                YES
    Xen:                     NO
    Xen VBD Error Tracking:  NO
Exporters:
    AWS Kinesis:             NO
    GCP PubSub:              NO
    MongoDB:                 YES
    Prometheus Remote Write: YES
```

### Additional info

Temporary solution:

```cmd
#enter docker containter
docker exec -it netdata /bin/bash

#install package
apk add apcupsd
```

**Comment by Ferroin** [comment]
To clarify, does installing apcupsd actually allow the plugin to work correctly? I was under the impression that it needs hardware access in a way that we can’t provide from inside a Docker container.

**Comment by misaka-eth** [comment]
Installing `apcupsd` allow the plugin to work correctly, but host must also have `apcupsd` installed and service `apcupsd.service` must be running.

my configuration in `charts.d/apcupsd.conf`:
```
# add all your APC UPSes in this array - uncomment it too
declare -A apcupsd_sources=(
    ["local"]="172.17.0.1:3551"
)
```
![image](https://user-images.githubusercontent.com/98682539/157461355-19ec620a-bb4a-4810-96f2-809d66784492.png)

**Comment by Ferroin** [comment]
I’ve created a PR to add apcupsd to our default Docker images (and clean up our other dependencies in the Docker images as well, because we’ve needed to tidy things up there for a while now). It should hopefully be merged some time early next week (if it doesn’t get approvals today, we’re likely to hold off merging until Monday), at which point the next nightly build will include these changes. I’ll be making sure it gets merged before the next stable release.

**Comment by F0x06** [comment]
Hi, Its seems that the `apcaccess` binary is not found on the `latest` version (`44f9556882fda3c4e56f793052988e30d2553c6fa278f11fe4756899a09c3fee`) again @Ferroin. I tried to mount the binary as a workaround, however the plugin is still trying to reach `localhost:3551` despite my `config/charts.d/apcupsd.conf` file:

```
# add all your APC UPSes in this array - uncomment it too
declare -A apcupsd_sources=(
    ["local"]="192.168.5.2:3551"
)
```

Telle me if i need to open a separate issue,

Regards,
F0x

**Comment by Steve-Brule** [comment]
Same issue as the comment above, it appears as though my config is ignored. Netdata keeps trying to reach apcupsd at 127.0.0.1, despite the config stating another host.

**Comment by ilyam8** [comment]
@F0x06 There is no `apcaccess` binary in the docker image, you need to [install it](https://learn.netdata.cloud/docs/installing/docker#adding-extra-packages-at-runtime).

**Comment by ilyam8** [comment]
> config/charts.d/apcupsd.conf

What is that directory? User config files are in `/etc/netdata/`. Should be `/etc/netdata/charts.d/apcupsd.conf`.

**Comment by F0x06** [comment]
> > config/charts.d/apcupsd.conf
> 
> What is that directory? User config files are in `/etc/netdata/`. Should be `/etc/netdata/charts.d/apcupsd.conf`.

It's the host mount path of my docker image, put the wrong one.

**Comment by F0x06** [comment]
@ilyam8 `NETDATA_EXTRA_DEB_PACKAGES` worked :), however my config file is still ignored, inside the docker container the file is placed at `/etc/netdata/charts.d/apcupsd.conf`, with this content:

```
# add all your APC UPSes in this array - uncomment it too
declare -A apcupsd_sources=(
    ["local"]="192.168.5.2:3551"
)
```

And i still get errors trying to reach `127.0.0.1`

`netdata  | 2024-02-05 16:47:46: charts.d.plugin: : apcupsd: command 'apcaccess status 127.0.0.1:3551 ' failed with code 1:`

**Comment by ilyam8** [comment]
I tested and found that loading user/stock configurations doesn't work. Fixed in #16939.
